Common/AR/source/ArcGISArViewRenderer.cpp

```diff
@@ -23,23 +23,17 @@
 using namespace Esri::ArcGISRuntime::Toolkit::Internal;
 
 /*!
+  \class ArcGISArViewRenderer
   \internal
- *
- * This class renders the passthrough camera image into the OpenGL frame.
+  \brief Renders the passthrough camera image into the OpenGL frame.
  */
 
-/*!
-  \internal
- */
```

Comment:
Why is this *not* needed? If it's solely to appease the doc build that's fine, I just want to understand why.

Reply:
The only time you need to use `\internal` is when QDoc wants to document something that should not appear in the public API doc. If you mark something `\internal` and QDoc cannot find the item to doc, QDoc complains.